### DISCUSSION TITLE
Opening folders directly on Flow launcher

### DISCUSSION THREAD

**Discussion by vidicius** [Flow-Launcher/Flow.Launcher#1225] [Q&A] [answered]
Hi guys,

Wanted to ask what am I doing wrong.

I used to look up a folder and simply hitting enter opened up that folder directly in Flow launcher search results, which is my desired outcome I love, but recently the enter just launches File explorer again.

I tried to check the settings or hit the enter with various modulators but no luck. Any help would be greatly appreciated.

Thanks so much.

**Comment by jjw24** [comment, marked as answer]
I think you are opening that result with Everything plugin instead of Explorer. You can check by going to the context menu and the last entry will tell you which plugin will be used.

If you have Everything and Explorer plugin running at the same time, you should see duplicate results for that folder. If you can't find the result to open with Explorer plugin, maybe your windows indexing is not configured for that folder for some reason, which you will need to take a look.

**Reply by vidicius** [reply]
Jop, duplicate folders were the issue. Thanks so much for frankly immediate response.

**Comment by jjw24** [comment]
Just going to leave a note here in case anyone else might need to know or ask, there is work going on to merge the behaviour of the two plugins so you can also open folders inside flow with Everything plugin.

**Reply by TheBestPessimist** [reply]
This was finished in V1.10: https://github.com/Flow-Launcher/Flow.Launcher/releases/tag/v1.10.0
